run/websockets/app.js

```diff
@@ -0,0 +1,102 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+const {redisClient, getRoomFromCache, addMessageToCache} = require('./redis');
+const {addUser, getUser, deleteUser} = require('./users');
+const express = require('express');
```

Comment:
nit: I expect to see libraries before file includes. Maybe that's not as strong a convention as I expect so just flagging as a nit.